app/jobs/finish_abandoned_job.rb

```diff
@@ -0,0 +1,7 @@
+# The goal of this job is to re-sync sponsors to ensure
```

Comment:
This class should probably be renamed to `FinishAbandonedMentorDiscussionsJob` (add "Mentor") and the file needs to be named to match.